packages/siopv2-oid4vp-rp-rest-api/__tests__/agent.ts

```diff
@@ -27,6 +27,7 @@ import { IPresentationExchange, PresentationExchange } from '@sphereon/ssi-sdk.p
 import { CheckLinkedDomain } from '@sphereon/did-auth-siop'
 import { entraAndSphereonCompatibleDef, entraVerifiedIdPresentation } from './presentationDefinitions'
 import Debug from 'debug'
+import { createHash } from 'crypto'
```

Comment:
We already should have this in the SD-JWT plugin right?. Also since this is the REST API on top it should use the other oid4vp base plugin anyway right? So then I rather see this is an optional param, with a default in the siopv2-oid4vp-rp-auth SSI-SDK plugin

Reply:
Also is this the same hasher we use in the SD-JWT plugin to begin with?

Reply:
this is a test file and we're using the `createHash` from `crypto` in there too